protocol-designer/src/components/StepEditForm/fields/FlowRateField/__tests__/FlowRateField.test.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react'
+import { describe, it, vi, beforeEach } from 'vitest'
+import { fireEvent, screen } from '@testing-library/react'
+import { fixtureP100096V2Specs } from '@opentrons/shared-data'
+import { renderWithProviders } from '../../../../../__testing-utils__'
+import { i18n } from '../../../../../localization'
+import { getPipetteEntities } from '../../../../../step-forms/selectors'
+import { FlowRateField } from '../index'
+
+vi.mock('../../../../../step-forms/selectors')
+const render = (props: React.ComponentProps<typeof FlowRateField>) => {
+  return renderWithProviders(<FlowRateField {...props} />, {
+    i18nInstance: i18n,
+  })[0]
+}
+const mockMockId = 'mockId'
+describe('FlowRateField', () => {
+  let props: React.ComponentProps<typeof FlowRateField>
+
+  beforeEach(() => {
+    props = {
+      disabled: false,
+      flowRateType: 'aspirate',
+      volume: 100,
+      value: null,
+      name: 'flowRate',
+      tiprack: 'tipRack:opentrons_flex_96_tiprack_1000ul',
+      updateValue: vi.fn(),
+      onFieldBlur: vi.fn(),
+      onFieldFocus: vi.fn(),
+      pipetteId: mockMockId,
+    }
+    vi.mocked(getPipetteEntities).mockReturnValue({
+      [mockMockId]: {
+        name: 'p50_single_flex',
+        spec: {
+          liquids: fixtureP100096V2Specs.liquids,
+          displayName: 'mockPipDisplayName',
+        } as any,
+        id: mockMockId,
+        tiprackLabwareDef: [
+          {
+            parameters: {
+              loadName: 'opentrons_flex_96_tiprack_1000ul',
+              tipLength: 1000,
+            },
+            metadata: { displayName: 'mockDisplayName' },
+          } as any,
+        ],
+        tiprackDefURI: ['mockDefURI1', 'mockDefURI2'],
+      },
+    })
+  })
+  it('renders the flowRateInput and clicking on it opens the modal with all the text', () => {
+    render(props)
+    screen.getByText('Flow Rate')
+    fireEvent.click(screen.getByRole('textbox', { name: '' }))
```

Comment:
I think we don't need to use name here.
```suggestion
    fireEvent.click(screen.getByRole('textbox'))
```